docs/batch/batch_registration.md

```diff
@@ -0,0 +1,67 @@
+# Batch Registration
+
+To register a new batch, start by [navigating](batch_introduction.md#batch-navigation) 
+into the batch summary view.
+![batch_summary](images/batch_summary_no_batches.png)
+
+!!! info "Email Notification"
+    Upon successful batch registration, all project collaborators
+    will automatically receive an email with a link to the created batch.
```

Comment:
Are these people whose accounts have been added to the project and/or people who have been set as responsible / PI while creating the project? Maybe link the respective section.